packages/components/Checkbox/SPEC.md

```diff
@@ -256,14 +325,20 @@ A disabled `Checkbox` is non-interactive, disallowing the user to click/tap on i
 
 A hovered `Checkbox` changes styling to communicate that the user has placed a cursor above it. On win32, this will show a checkmark in a checkbox.
 
+Note: Not Supported on Android
+
 #### Focused state
 
 A focused `Checkbox` changes styling to communicate that the user has placed keyboard focus on it. This styling is usually the same to the one in the hovered state plus extra styling on the outline to indicate keyboard focus has been placed on the component.
 
+Note: Not Supported on Android
+
 #### Pressed state
 
 A pressed `Checkbox` changes styling to communicate that the user is currently pressing it.
 
+Note: Not Supported on Android
```

Comment:
This isn't supported an Android? Why not? Wouldn't that be conveyed via the ripple?

Reply:
Addressed on https://github.com/microsoft/fluentui-react-native/pull/2416